docs/extend/manifest.md

```diff
@@ -0,0 +1,239 @@
+<!--[metadata]>
++++
+aliases = [
+"/engine/extend/"
+]
+title = "Plugin manifest"
+description = "How develop and use a plugin with the managed plugin system"
+keywords = ["API, Usage, plugins, documentation, developer"]
+advisory = "experimental"
+[menu.main]
+parent = "engine_extend"
+weight=1
++++
+<![end-metadata]-->
+
+# Plugin manifest version 0
```

Comment:
Docs style is to just capitalize the first word as in a sentence except in the case of proper names. I think the current usage is correct.